### PR TITLE
replace left pane and Commit blank pages for each docs pages

### LINKED ISSUE #9: mvp4 - Create a top navigation pane, similar to https://segment.com/docs/
https://github.com/Shippable/pm/issues/8293

### LINKED ISSUE #10: mvp4 - Add the left navigation pane with up to 2 levels. Headings in the left pane can be expanded or collapsed. Clicking on any subheading opens an html page.
https://github.com/Shippable/pm/issues/8293

### LINKED ISSUE #11: mvp5 - Replace top navigation, left pane heading, and left pane subheading with content from here
https://github.com/Shippable/pm/issues/8293

### LINKED ISSUE #12: mvp5 - Create a blank page for each sub-heading with the page name from the Page Name column.
https://github.com/Shippable/pm/issues/8293

### PR DESCRIPTION
**Pull request by 0x-x0**
closes #9 
closes #10 
closes https://github.com/Shippable/docs/issues/12
closes https://github.com/Shippable/docs/issues/11

Merge mvp 4 and 5 to test out the links
anim

#### This is how it looks on my mobile screen 
img_0818

### CODE REVIEW

File: theme/saas/static/css/shippable.css [hunk 1]
```diff
@@ -142,3 +142,289 @@ blockquote {
...
+  margin: 0;
+  padding: 0
+}
+
+menu,ol,ul {
+  list-style: none
+}
+
+.shipContent {
+    display: -ms-flexbox;
```
Comment (shrivara) [inline theme/saas/static/css/shippable.css]: 2 spaces everywhere.

File: theme/saas/static/css/shippable.css [hunk 2]
```diff
@@ -142,3 +142,289 @@ blockquote {
...
+    -ms-flex-align: stretch;
+    align-items: stretch;
+    -ms-flex-direction: row-reverse;
+    flex-direction: row-reverse;
+    height: calc(100vh - 63px)
+}
+
+.shipContent,.shipContent-body {
+    overflow-y: auto;
+    overflow-x: hidden
```
Comment (shrivara) [inline theme/saas/static/css/shippable.css]: Semi-colon.

File: theme/saas/static/css/shippable.css [hunk 3]
```diff
@@ -142,3 +142,289 @@ blockquote {
   color: #333;
   border-left: 5px solid #ddd;
 }
+
```
Comment (shrivara) [inline theme/saas/static/css/shippable.css:145]: We should run this through some kind of css lint. There are few missing semi-colons.

File: theme/saas/static/js/expandCollapse.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+$(document).ready(function () {
```
Comment (shrivara) [inline theme/saas/static/js/expandCollapse.js]: `function` in next line.

**Review by shrivara — CHANGES_REQUESTED** [review]

**Comment by 9ok3r** [comment]
This is how it looks on my mobile screen 
img_0818

**Review by shrivara — APPROVED** [review]

**Pull request merged**